karabo_data/geometry2.py

```diff
@@ -378,9 +377,9 @@ def _plotting_dimensions(self):
         min_yx = corners.min(axis=0) - 20
```

Comment:
Do we want to keep these margins?

Reply:
Possibly not. I think some visual margin is useful for displaying the data, but making the array bigger than necessary is probably not the right way to achieve that.